### DISCUSSION TITLE
Fetch results from specific sources

### DISCUSSION THREAD

**Discussion by arda618** [searxng/searxng#2373] [Q&A] [answered]
I want my instance to get results only from the websites of my wish (what I mean is NOT an engine, but a website that contains searchable data). Let's say I only have Google enabled and I only want a list of websites to be indexed, and ignore all the other results that are not from the websites that I've listed. What do I have to do to achieve that? I have tried to get ChatGPT to help me but the solution it has provided doesn't seem to work.

**Comment by return42** [comment, marked as answer]
You can disable search engines (by default) in your `/etc/searx/settings.yml` --> https://docs.searxng.org/admin/engines/settings.html#engine-settings see `disabled: true`.

> and I only want a list of websites to be indexed, and ignore all the other results that are not from the websites that I've listed. 

I think this is not possible by configuration / may be there is a solution with hostname_replace but I haven't tried.
